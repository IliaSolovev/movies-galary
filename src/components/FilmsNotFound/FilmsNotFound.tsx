import React from "react";
import s from './filmsNotFound.module.scss';
interface Props {

}

const FilmsNotFound: React.FC<Props> = (props) => {
    return (
        <h2 className={s.title}>
        Films Not Found
        </h2>
    )
};

export default FilmsNotFound