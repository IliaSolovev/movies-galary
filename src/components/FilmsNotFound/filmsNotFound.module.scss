@import "../../styles/variables";
.title{
  width: 1000vw;
  height: 60vh;
  font-family: $mainFont;
  font-size: 2em;
  text-align: center;
}