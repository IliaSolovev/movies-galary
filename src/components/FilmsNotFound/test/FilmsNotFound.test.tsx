import React from "react";
import { shallow } from "enzyme";
import FilmsNotFound from "../FilmsNotFound";
import toJson from "enzyme-to-json";

it("should render films not found", () => {
	const component = shallow(<FilmsNotFound/>);
	expect(toJson(component)).toMatchSnapshot();
});