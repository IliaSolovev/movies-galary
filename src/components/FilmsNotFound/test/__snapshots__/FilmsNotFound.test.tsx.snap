// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render films not found 1`] = `
<h2
  className="title"
>
  Films Not Found
</h2>
`;
